.github/workflows/snyk-analysis.yml

```diff
@@ -0,0 +1,45 @@
+name: "Snyk"
```

Comment:
@abstractj can you review this one please?

Reply:
LGTM